canonical/jp.ubuntu.com
# WD-11650 - refresh /iot

## Done

Refreshed /iot page

## QA

- [demo link](https://jp-ubuntu-com-486.demos.haus/iot)
- [copy doc](https://docs.google.com/document/d/1kKk7sRq21je9sJC93-6JjTqhNbR-Eh3WRBCJqH44i00/edit)
- [figma](https://www.figma.com/design/tI8YK4wQnKiWLAxdX1fWnB/24.10-jp.ubuntu.com?node-id=342-7360&t=1eMNv3L03JlwfX6u-0)
- View the site on demo
- Run through the following [QA steps](https://discourse.canonical.com/t/qa-steps/152)
- Check that the page is correct according to figma and copy doc

## Issue / Card
[WD-11650](https://warthogs.atlassian.net/browse/WD-11650)

[WD-11650]: https://warthogs.atlassian.net/browse/WD-11650?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
